Modules/resource.c

```diff
@@ -340,155 +340,175 @@ resource_methods[] = {
 /* Module initialization */
 
 
-static struct PyModuleDef resourcemodule = {
-    PyModuleDef_HEAD_INIT,
-    "resource",
-    NULL,
-    -1,
-    resource_methods,
-    NULL,
-    NULL,
-    NULL,
-    NULL
-};
-
-PyMODINIT_FUNC
-PyInit_resource(void)
+static int resource_exec(PyObject *module)
```

Comment:
PEP 7 requires to write it as:

```suggestion
static int
resource_exec(PyObject *module)
```

Reply:
thanks, done